config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Osra::Application.routes.draw do
 
+  devise_for :admin_users, path: "hq", as: "hq", path_names: { sign_in: 'login', sign_out: 'logout' },
+                           :controllers => { :sessions => "hq/devise/sessions" }
   namespace :hq do
+    root to: "sponsors#index"                 #should be changed to "dashboard" when implemented"
```

Comment:
Instead of a comment here, I'd recommend a spec marked pending.  When it passes, it will notify you to fix it.  You can put this comment inside the routing spec for the dashboard view.